docs/src/main/sphinx/admin/event-listeners-openlineage.md

```diff
@@ -210,6 +210,10 @@ event-listener.config-files=etc/openlineage-event-listener.properties,...
     - List of custom url params to be added to final HTTP Request. See
     [](openlineage-event-listener-custom-url-params) for more details.
     - Empty
+*
+    - openlineage-event-listener.transport.compression
+    - Compression codec used for reducing size of HTTP body. Allowed values: ``none``, ``gzip``.
+    - ``none``
```

Comment:
Why do we avoid using `gzip` by default?

Reply:
OpenLineage is a JSON specification + Java/Python clients. But there is a bunch of servers which accept OpenLineage events JSON via HTTP, Marquez is just one of them. Other servers may not support GZIP, and we don't know which server implementations used by most of users.

Reply:
Ok

Reply:
Added